### ISSUE TITLE
Design evaluation setting and metrics

### ISSUE THREAD

**Issue by fedenanni** [Living-with-machines/TargetedSenseDisambiguation#7]
Our paper focuses around two [word sense disambiguation](https://en.wikipedia.org/wiki/Word-sense_disambiguation) (WSD) tasks:
- traditional WSD, where we have a word (say `machine`) in a sentence and we have to predict which sense is the most appropriate, given a predifined list of `machine` senses.
- extended WSD, where we have a word (say `locomotive`) which is related to (at least one) sense of another word (say `machine`) and the task is to predict which sense of `machine` is more appropriate, given the word `locomotive` in a sentence. The relation could be a `synonym`, an `hyponym` etc.

**Goal:** to define a clear evaluation setting (train/test splitting) and evaluation metrics

**TLDR** (main points from the discussion below - to be updated):

- do we want a single evaluation framework or two different ones? [[to know more](https://github.com/Living-with-machines/HistoricalDictionaryExpansion/issues/7#issuecomment-732943680)]
- do we have examples in the literature of our "extended WSD"? How is it called / evaluated? possible answer: look into lexical substitution and presudowords in wsd [[to know more](https://github.com/Living-with-machines/HistoricalDictionaryExpansion/issues/7#issuecomment-733028399)]
- do we want to write a broad diachronic WSD paper or a piece focused on the different senses of the lemma "machine" across time and sources? [[to know more](https://github.com/Living-with-machines/HistoricalDictionaryExpansion/issues/7#issuecomment-733168710)]

Currently blocked by #2
